### PR TITLE
Users/ajya/typo corrections tasks

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/task.json [hunk 1]
```diff
@@ -402,7 +402,7 @@
         "Failedtodeploywebapppackageusingkuduservice": "Failed to deploy App Service package using kudu service : %s",
         "Unabletodeploywebappresponsecode": "Unable to deploy App Service due to error code : %s",
         "MSDeploygeneratedpackageareonlysupportedforWindowsplatform": "MSDeploy generated packages are only supported for Windows platform.",
-        "UnsupportedinstalledversionfoundforMSDeployversionshouldbealteast3orabove": "Unsupported installed version: %s found for MSDeploy. version should be alteast 3 or above",
+        "UnsupportedinstalledversionfoundforMSDeployversionshouldbeatleast3orabove": "Unsupported installed version: %s found for MSDeploy. version should be at least 3 or above",
```
Comment (rajatagrawal-dev) [inline Tasks/AzureRmWebAppDeployment/task.json]: minor: other places you used "atleast" while here it is "at least" for the same string.

Comment (Ajay-MS) [inline Tasks/AzureRmWebAppDeployment/task.json]: updated

File: Tasks/AzureAppServiceManage/task.json [hunk 2]
```diff
@@ -239,8 +239,8 @@
         "FailedtoRestartAppService": "Failed to restart App Service for '%s'. Response Code : %s (%s)",
         "RestartAppServiceAccepted": "Restart request accepted by App Service : %s",
         "InvalidAction": "Invalid Action selected !",
-        "WARNINGCannotupdatedeploymentstatusSCMendpointisnotenabledforthiswebsite": "WARNING: Cannot update deployment statu. SCM endpoint is not enabled for this website.",
-        "CouldnotfetchacccesstokenforAzureStatusCode": "Could not fetch acccess token for Azure App Service. StatusCode %s (%s)",
+        "WARNINGCannotupdatedeploymentstatusSCMendpointisnotenabledforthiswebsite": "WARNING: Cannot update deployment status. SCM endpoint is not enabled for this website.",
+        "CouldnotfetchaccesstokenforAzureStatusCode": "Could not fetch acccess token for Azure App Service. StatusCode %s (%s)",
```
Comment (kmkumaran) [inline Tasks/AzureAppServiceManage/task.json]: What is the change for this resource key? I see bunch of resjson having just this change...

File: Tasks/AzureCloudPowerShellDeployment/task.json [hunk 3]
```diff
@@ -134,7 +134,7 @@
             "label": "Diagnostic storage account keys",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "Provide storage keys for diagnostics storage account in Role:Storagekey format. The diagnostics storage account name for each role will be obtained from diagnostics config file (.wadcfgx). If the .wadcfgx file for a role is not found, diagnostics extensions won’t be set for the role. If the storage account name is missing in the .wadcfgx file, the default storage account will be used for storing diagnostics results and the storage key parameters from deployment task will be ignored. It’s recommended to save <storage_account_key> as a secret variable unless there is no sensitive information in the diagnostics result for your environment. <br/><br/>For example,<br/> WebRole: &lt;WebRole_storage_account_key&gt;<br/>WorkerRole: &lt;WorkerRole_stoarge_accoubt_key&gt;",
```
Comment (kmkumaran) [inline Tasks/AzureCloudPowerShellDeployment/task.json]: I don't see any change here.

Comment (Ajay-MS) [inline Tasks/AzureCloudPowerShellDeployment/task.json]: accoubt is changed to account

File: Tasks/AzureAppServiceManage/task.json (hunk 2 above)
Comment (Ajay-MS) [inline Tasks/AzureAppServiceManage/task.json]: In this key contains acccess instead of access.

**Pull request merged**